packages/dialtone-vue2/recipes/leftbar/contact_centers_row/contact_centers_row_default.story.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <dt-recipe-contact-centers-row
+    :description="$attrs.description"
+    :unread-count="$attrs.unreadCount"
+    :aria-label="$attrs.ariaLabel"
+    :selected="$attrs.selected"
+    @click="$attrs.onClick"
+    @click-menu="$attrs.clickMenu"
+  />
```

Comment:
Check other stories for guidance on how to add slots here so the user can pass content to them.

Reply:
I've follow the example from `recipes/notices/top_banner_info/top_banner_info_default.story.vue`, but I'm not sure it's really the right way. Let me know if you can suggest another component to use for reference.